plugins/gcloud/lib/samson_gcloud/image_tagger.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 module SamsonGcloud
   class ImageTagger
+    PRODUCTION_TAG = 'production'
```

Comment:
can we freeze this??

Reply:
Isn't that taken care of with `frozen_string_literal`?

Reply:
hmmm, got confused on Constant, You are right!!